docs_src/microservices/device/supported/device-usb-camera/supplementary-info/USB-protocol.md

```diff
@@ -0,0 +1,9 @@
+# USB Camera Device Service Specifications
+
+## USB Protocol Properties
+| Property | Description | EdgeX Value Type |               
+| -- | -- | -- |  
+| **Paths** | A list of internal [/dev/video paths](https://www.kernel.org/doc/html/v4.9/media/uapi/v4l/dev-capture.html) for the camera device. This list includes all streaming capable video paths for each device. | Object |  
+| **SerialNumber** | The serial number of the camera device. | String |  
+| **CardName** | The manufacturer name of the camera device. | String |
```

Comment:
This wording makes it sound like it is the name of the mfg. The field in use is `ID_V4L_PRODUCT`, so maybe something like:

```suggestion
| **CardName** | The manufacturer specified product name of the camera device. | String |
```
or just
```suggestion
| **CardName** | The manufacturer specified name of the camera device. | String |
```